packages/components/Notification/src/__tests__/__snapshots__/Notification.test.tsx.snap

```diff
@@ -107,8 +107,6 @@ exports[`Notification component tests Notification default 1`] = `
             "fontFamily": "System",
             "fontSize": 15,
             "fontWeight": "400",
-            "letterSpacing": -0.24,
-            "lineHeight": 20,
```

Comment:
Are you sure these aren't necessary? I thought we had these because of the design spec?

Reply:
Updated the description above to explain why this snapshot changed (similar change was made [here](https://github.com/microsoft/fluentui-react-native/pull/2472/files))

Reply:
Thanks! If these are being set elsewhere, then alright